### DISCUSSION TITLE
求问怎么在docker部署？

### DISCUSSION THREAD

**Discussion by mrchenxxx** [Mrs4s/go-cqhttp#1386] [Q&A] [answered]
求问怎么在docker部署？

**Comment by LainNetWork** [comment]
自己写个dockerFile不就行了

**Comment by qianjunakasumi** [comment, marked as answer]
你可选择已经构建好的内容：
https://github.com/Mrs4s/go-cqhttp/pkgs/container/go-cqhttp

或者自行修改 https://github.com/Mrs4s/go-cqhttp/blob/master/Dockerfile 构建镜像
